plugins/source/azure/resources/services/storage/queue_access_policy.go

```diff
@@ -37,14 +38,34 @@ func fetchQueueACL(ctx context.Context, meta schema.ClientMeta, parent *schema.R
 	queue := parent.Item.(*armstorage.ListQueue)
 	acc := parent.Parent.Item.(*armstorage.Account)
 
+	storageKey, err := cl.GetStorageAccountKey(ctx, acc)
+	if err == client.ErrNoStorageKeysFound || storageKey == "" {
+		var lgr *zerolog.Event
+		if err != nil && err != client.ErrNoStorageKeysFound {
+			lgr = cl.Logger().Error().Err(err)
```

Comment:
shouldn't we also ret the err here?

Reply:
We did below, but simplified in [`d4a690e` (#10554)](https://github.com/cloudquery/cloudquery/pull/10554/commits/d4a690e09f529d2b2c1e4db3d27fcf720fe17a06)